registry/service.go

```diff
@@ -40,7 +36,14 @@ func (s *Service) Auth(authConfig *cliconfig.AuthConfig) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	endpoint, err := NewEndpoint(index, nil)
+
+	endpointVersion := APIVersion(APIVersionUnknown)
```

Comment:
I don't think this cast is necessary, is it? ApiVersionUnknown is already an APIVersion. Same in line 63.